hugo/content/info/contrib.md

```diff
@@ -50,6 +48,7 @@ Please note the following important dates.
 ### Register your meeting
 
 If you are a conference publications chair, you must register your intention to submit your proceedings.
+(Workshop chairs should do this     
```

Comment:
incomplete sentence

Reply:
I seem unable to write anything without glaring typographical and other errors.